lib/backburner.js

```diff
@@ -73,6 +79,48 @@ Backburner.prototype = {
     }
   },
 
+  _trigger: function(eventName /* ...args */) {
+    var callbacks = this._eventCallbacks[eventName];
+    if (callbacks) {
+      var len = arguments.length;
```

Comment:
how about we just support 1 arg to the callback, this would simplify this code, and really wont hurt the consumer.